src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -61,23 +62,26 @@ function _handleUnsupportedToken({
   error: unknown
   addUnsupportedToken: (params: AddGpUnsupportedTokenParams) => void
 }) {
-  if (_isValidApiError(error)) {
+  if (_isValidOperatorError(error)) {
     // Unsupported token
-    if (error.errorType === ApiErrorCodes.UnsupportedToken) {
+    if (error.type === ApiErrorCodes.UnsupportedToken) {
```

Comment:
not important, but I think `_handleUnsupportedToken` should only have the code of this if's branch (67-77). What you are doing in the method right now, is handle any errors, not just validOperatorError

Also, isValidOperatorError, or isInvalidOperatorError?